soc/nordic/nrf54h/bicr/bicrgen-schema.json

```diff
@@ -0,0 +1,422 @@
+            "Unconfigured, system will not boot",
+            "VDDH supplied with 2.1-5.5V and VDD regulated by the chip, inductor present",
+            "Both VDD and VDDH supplied with 1.8V, inductor present"
+          ],
+          "enum": [
+            "UNCONFIGURED",
+            "VDD_VDDH_1V8",
+            "VDDH_2V1_5V5"
+          ],
+          "default": "UNCONFIGURED"
+        }
+      },
+      "required": [
+        "scheme"
+      ]
+    },
+    "ioPortPower": {
+      "type": "object",
+      "title": "IO port power configuration",
+      "properties": {
+        "p1Supply": {
+          "type": "string",
+          "title": "P1 power supply (VDDIO_P1)",
+          "enumNames": [
+            "Not supplied (P1 not used)",
+            "VDDIO_P1 shorted to VDD",
+            "VDDIO_P1 connected to an external 1.8V supply or to VDD_EXT"
+          ],
+          "enum": [
+            "DISCONNECTED",
+            "SHORTED",
+            "EXTERNAL_1V8"
+          ],
+          "default": "DISCONNECTED"
+        },
+        "p2Supply": {
+          "type": "string",
+          "title": "P2 power supply (VDDIO_P2)",
+          "enumNames": [
+            "Not supplied (P2 not used)",
+            "VDDIO_P2 shorted to VDD",
+            "VDDIO_P2 connected to an external 1.8V supply or to VDD_EXT"
+          ],
+          "enum": [
+            "DISCONNECTED",
+            "SHORTED",
+            "EXTERNAL_1V8"
+          ],
+          "default": "DISCONNECTED"
+        },
+        "p6Supply": {
+          "type": "string",
+          "title": "P6 power supply (VDDIO_P6)",
+          "enumNames": [
+            "Not supplied (P6 not used)",
+            "VDDIO_P6 shorted to VDD",
+            "VDDIO_P6 connected to an external 1.8V supply or to VDD_EXT"
+          ],
+          "enum": [
+            "DISCONNECTED",
+            "SHORTED",
+            "EXTERNAL_1V8"
+          ],
+          "default": "DISCONNECTED"
+        },
+        "p7Supply": {
+          "type": "string",
+          "title": "P7 power supply (VDDIO_P7)",
+          "enumNames": [
+            "Not supplied (P7 not used)",
+            "VDDIO_P7 shorted to VDD",
+            "VDDIO_P7 connected to an external 1.8V supply or to VDD_EXT"
+          ],
+          "enum": [
+            "DISCONNECTED",
+            "SHORTED",
+            "EXTERNAL_1V8"
+          ],
+          "default": "DISCONNECTED"
+        },
+        "p9Supply": {
+          "type": "string",
+          "title": "P9 power supply (VDDIO_P9)",
+          "enumNames": [
+            "Not supplied (P9 not used)",
+            "VDDIO_P9 shorted to VDD",
+            "VDDIO_P9 connected to an external 1.8V supply or to VDD_EXT",
+            "VDDIO_P9 connected to an external 1.8V-3V supply"
+          ],
+          "enum": [
+            "DISCONNECTED",
+            "SHORTED",
+            "EXTERNAL_1V8",
+            "EXTERNAL_FULL"
+          ],
+          "default": "DISCONNECTED"
```

Comment:
EXTERNAL_FULL needs to be default. DISCONNECTED is not safe with more than 1.8V at VDDIO_P9.